helm/boot.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+mkdir ~/.funcx
+mkdir ~/.funcx/$1
+mkdir ~/.funcx/credentials
+cp /funcx/config/config.py ~/.funcx
+cp /funcx/$1/* ~/.funcx/$1
+cp /funcx/credentials/* ~/.funcx/credentials
+if [ -z "$2" ]; then
+  funcx-endpoint start $1
+else
+  funcx-endpoint start $1 --endpoint-uuid $2
+fi
+
+sleep infinity
```

Comment:
I have a script that can check funcx-endpoint process aliveness. Think it is better to replace `sleep infinity` here?
```
while pgrep funcx-endpoint >/dev/null;
    do
        echo "funcx-endpoint process is still alive. Next check in 600s."
        sleep 600;
    done
echo "funcx-endpoint process exited. Restarting endpoint"
```

Reply:
Just want the endpoint pod to restart if the process already exits